### DISCUSSION TITLE
Spa in error

### DISCUSSION THREAD

**Discussion by Bleygo** [Yogui79/IntexPureSpa#61] [Q&A]
Bonjour,
Mon spa est un 28452EX, version 2022 avec console sans fil aussi, besoin de changer quelques choses dans le code ou pas pour que ce soit ok ?

**Comment by Yogui79** [comment]
Bonjour,
a priori le protocole est différent je n'ai malheureusement pas plus d'info , mais si vous avez quelques notions de programmation par ex on peu essayer de décoder le protocole

**Reply by Bleygo** [reply]
Merci pour le retour.
Oui, j'ai quelques notions de programmation, c et c++ mais ça date ;).
Et le plus important, la motivation est là !
Donc que puis-je faire ?

**Reply by Katoche690** [reply]
Bonjour, 
@Yogui79 je n'ai pas de compétence spécial mais tu peux peut etre me guider ? Si la tablette suffit je peux aussi te l'envoyer si ca te vas ? 
Merci pour ton travail en tout cas

**Comment by Bleygo** [comment]
Si ça peut aider, mes logs mais ça ne clignote jamais rapidement :
14:04:03.557 -> Channel 0
14:04:06.048 -> Channel 1
14:04:08.518 -> Channel 2
14:04:11.057 -> Channel 3
14:04:13.555 -> Channel 4
14:04:16.047 -> Channel 5
14:04:18.552 -> Channel 6
14:04:21.074 -> Channel 7
14:04:23.564 -> Channel 8
14:04:26.072 -> Channel 9
14:04:28.546 -> Channel A
14:04:31.064 -> Channel B
14:04:33.556 -> Channel C
14:04:36.048 -> Channel D
14:04:38.542 -> Channel E
14:04:41.049 -> Channel F
14:04:43.545 -> Channel 10
14:04:46.073 -> Channel 11
14:04:48.543 -> Channel 12
14:04:51.072 -> Channel 13
14:04:53.566 -> Channel 14
14:04:56.043 -> Channel 15
14:04:58.567 -> Channel 16
14:05:01.028 -> Channel 17
14:05:03.574 -> Channel 18
14:05:06.045 -> Channel 19
14:05:08.565 -> Channel 1A
14:05:11.069 -> Channel 1B
14:05:13.568 -> Channel 1C
14:05:16.072 -> Channel 1D
14:05:18.592 -> Channel 1E
14:05:21.093 -> Channel 1F
14:05:23.598 -> Channel 20
14:05:26.107 -> Channel 21
14:05:28.597 -> Channel 22
14:05:31.092 -> Channel 23
14:05:33.580 -> Channel 24
14:05:36.087 -> Channel 25
14:05:38.579 -> Channel 26
14:05:41.101 -> Channel 27
14:05:43.603 -> Channel 28
14:05:46.099 -> Channel 29
14:05:48.598 -> Channel 2A
14:05:53.600 -> Channel 2C
14:05:56.101 -> Channel 2D
14:05:58.594 -> Channel 2E
14:06:01.098 -> Channel 2F
14:06:03.595 -> Channel 30
14:06:06.106 -> Channel 31
14:06:08.610 -> Channel 32
14:06:11.089 -> Channel 33
14:06:13.600 -> Channel 34
14:06:16.088 -> Channel 35
14:06:18.569 -> Channel 36
14:06:21.115 -> Channel 37
14:06:23.619 -> Channel 38
14:06:26.104 -> Channel 39
14:06:28.631 -> Channel 3A
14:06:31.091 -> Channel 3B
14:06:33.632 -> Channel 3C
14:06:36.119 -> Channel 3D
14:06:38.624 -> Channel 3E
14:06:41.124 -> Channel 3F
14:06:43.610 -> Channel 40
14:06:46.135 -> Channel 41
14:06:48.626 -> Channel 42
14:06:51.131 -> Channel 43
14:06:53.607 -> Channel 44
14:06:56.098 -> Channel 45
14:06:58.605 -> Channel 46
14:07:01.132 -> Channel 47
14:07:03.608 -> Channel 48
14:07:06.121 -> Channel 49
14:07:08.613 -> Channel 4A
14:07:11.089 -> Channel 4B
14:07:13.630 -> Channel 4C
14:07:16.121 -> Channel 4D
14:07:18.633 -> Channel 4E
14:07:21.124 -> Channel 4F
14:07:23.662 -> Channel 50
14:07:26.150 -> Channel 51
14:07:28.657 -> Channel 52
14:07:31.163 -> Channel 53
14:07:33.648 -> Channel 54
14:07:36.160 -> Channel 55
14:07:38.655 -> Channel 56
14:07:41.157 -> Channel 57
14:07:43.662 -> Channel 58
14:07:46.164 -> Channel 59
14:07:48.665 -> Channel 5A
14:07:51.157 -> Channel 5B
14:07:53.646 -> Channel 5C
14:07:56.158 -> Channel 5D
14:07:58.670 -> Channel 5E
14:08:01.168 -> Channel 5F
14:08:03.662 -> Channel 60
14:08:06.141 -> Channel 61
14:08:08.649 -> Channel 62
14:08:11.139 -> Channel 63
14:08:13.664 -> Channel 64
14:08:16.154 -> Channel 65
14:08:18.649 -> Channel 66
14:08:21.182 -> Channel 67
14:08:23.678 -> Channel 68
14:08:26.179 -> Channel 69
14:08:28.703 -> Channel 6A
14:08:31.193 -> Channel 6B
14:08:33.691 -> Channel 6C
14:08:36.186 -> Channel 6D
14:08:38.690 -> Channel 6E
14:08:41.194 -> Channel 6F
14:08:43.710 -> Channel 70
14:08:46.201 -> Channel 71
14:08:48.699 -> Channel 72
14:08:51.177 -> Channel 73
14:08:53.703 -> Channel 74
14:08:56.176 -> Channel 75
14:08:58.696 -> Channel 76
14:09:01.202 -> Channel 77
14:09:03.688 -> Channel 78
14:09:06.180 -> Channel 79
14:09:08.681 -> Channel 7A
14:09:11.188 -> Channel 7B
14:09:13.690 -> Channel 7C
14:09:16.196 -> Channel 7D
14:09:18.686 -> Channel 7E
14:09:21.177 -> Channel 7F
14:09:23.718 -> Channel 80
14:09:26.198 -> Channel 81
14:09:28.693 -> Channel 82
14:09:31.211 -> Channel 83
14:09:33.705 -> Channel 84
14:09:36.209 -> Channel 85
14:09:38.700 -> Channel 86
14:09:41.216 -> Channel 87
14:09:43.702 -> Channel 88
14:09:46.200 -> Channel 89
14:09:48.698 -> Channel 8A
14:09:51.210 -> Channel 8B
14:09:53.713 -> Channel 8C
14:09:56.186 -> Channel 8D
14:09:58.704 -> Channel 8E
14:10:01.189 -> Channel 8F
14:10:03.696 -> Channel 90
14:10:06.190 -> Channel 91
14:10:08.691 -> Channel 92
14:10:11.185 -> Channel 93
14:10:13.713 -> Channel 94
14:10:16.202 -> Channel 95
14:10:18.691 -> Channel 96
14:10:21.191 -> Channel 97
14:10:23.693 -> Channel 98
14:10:26.200 -> Channel 99
14:10:28.705 -> Channel 9A
14:10:31.218 -> Channel 9B
14:10:33.709 -> Channel 9C
14:10:36.169 -> Channel 9D
14:10:38.703 -> Channel 9E
14:10:41.189 -> Channel 9F
14:10:43.714 -> Channel A0
14:10:46.191 -> Channel A1
14:10:48.698 -> Channel A2
14:10:51.206 -> Channel A3
14:10:53.711 -> Channel A4
14:10:56.196 -> Channel A5
14:10:58.687 -> Channel A6
14:11:01.207 -> Channel A7
14:11:03.714 -> Channel A8
14:11:06.209 -> Channel A9
14:11:08.683 -> Channel AA
14:11:11.204 -> Channel AB
14:11:13.686 -> Channel AC
14:11:16.194 -> Channel AD
14:11:18.705 -> Channel AE
14:11:21.200 -> Channel AF
14:11:23.706 -> Channel B0
14:11:26.192 -> Channel B1
14:11:28.710 -> Channel B2
14:11:31.199 -> Channel B3
14:11:33.700 -> Channel B4
14:11:36.193 -> Channel B5
14:11:38.708 -> Channel B6
14:11:41.211 -> Channel B7
14:11:43.712 -> Channel B8
14:11:46.195 -> Channel B9
14:11:48.705 -> Channel BA
14:11:51.200 -> Channel BB
14:11:53.706 -> Channel BC
14:11:56.205 -> Channel BD
14:11:58.702 -> Channel BE
14:12:01.206 -> Channel BF
14:12:03.700 -> Channel C0
14:12:06.190 -> Channel C1
14:12:08.708 -> Channel C2
14:12:11.183 -> Channel C3
14:12:13.698 -> Channel C4
14:12:16.191 -> Channel C5
14:12:18.707 -> Channel C6
14:12:21.199 -> Channel C7
14:12:23.710 -> Channel C8
14:12:26.202 -> Channel C9
14:12:28.702 -> Channel CA
14:12:31.204 -> Channel CB
14:12:33.711 -> Channel CC
14:12:36.204 -> Channel CD
14:12:38.712 -> Channel CE
14:12:41.199 -> Channel CF
14:12:43.705 -> Channel D0
14:12:46.191 -> Channel D1
14:12:48.703 -> Channel D2
14:12:51.210 -> Channel D3
14:12:53.705 -> Channel D4
14:12:56.197 -> Channel D5
14:12:58.699 -> Channel D6
14:13:01.213 -> Channel D7
14:13:03.700 -> Channel D8
14:13:06.185 -> Channel D9
14:13:08.718 -> Channel DA
14:13:11.198 -> Channel DB
14:13:13.701 -> Channel DC
14:13:16.182 -> Channel DD
14:13:18.689 -> Channel DE
14:13:21.187 -> Channel DF
14:13:23.707 -> Channel E0
14:13:26.180 -> Channel E1
14:13:28.704 -> Channel E2
14:13:31.207 -> Channel E3
14:13:33.694 -> Channel E4
14:13:36.200 -> Channel E5
14:13:38.685 -> Channel E6
14:13:41.189 -> Channel E7
14:13:43.693 -> Channel E8
14:13:46.181 -> Channel E9
14:13:48.684 -> Channel EA
14:13:51.200 -> Channel EB
14:13:53.691 -> Channel EC
14:13:56.191 -> Channel ED
14:13:58.698 -> Channel EE
14:14:01.190 -> Channel EF
14:14:03.695 -> Channel F0
14:14:06.192 -> Channel F1
14:14:08.706 -> Channel F2
14:14:11.186 -> Channel F3
14:14:13.707 -> Channel F4
14:14:16.195 -> Channel F5
14:14:18.683 -> Channel F6
14:14:21.163 -> Channel F7
14:14:23.692 -> Channel F8
14:14:26.186 -> Channel F9
14:14:28.702 -> Channel FA
14:14:31.191 -> Channel FB
14:14:33.685 -> Channel FC
14:14:36.192 -> Channel FD
14:14:38.689 -> Channel FE
14:14:41.194 -> Channel FFFFFFFF

**Comment by Bleygo** [comment]
Donc sur le spa, il y a bien une carte esp 32

![IMG_20230407_134123](https://user-images.githubusercontent.com/126673463/230609777-3c2868eb-5d65-4b56-92b9-e8c165dc87fe.jpg)

Et mon câblage semble ok

![IMG_20230407_143209](https://user-images.githubusercontent.com/126673463/230609861-6d9afbe0-3c84-415a-9f00-6a03f5961e78.jpg)

![IMG_20230407_143242](https://user-images.githubusercontent.com/126673463/230609916-09515dad-3fc0-43ae-a8fc-b7b842b693ee.jpg)

**Reply by Bleygo** [reply]
@Yogui79 , sympa ce que je viens de trouver : https://github.com/mathieu-mp/homeassistant-intex-spa
Adaptable sous Jeedom ?
